src/vocgui/models.py

```diff
@@ -92,10 +96,46 @@ class DocumentImage(models.Model):
     document = models.ForeignKey(Document,
                                  on_delete=models.CASCADE,
                                  related_name='document_image')
-
+    def save_original_img(self):
+        """
+        Function to save rough image with '_original' extension
+        """
+        name_elements = self.image.path.split('.')
+        for elem in name_elements:
+            if elem != name_elements[-1]:
+                new_path = elem + '_'
+        new_path = new_path + 'original.' + name_elements[-1]
+        img = Image.open(self.image.path)
+        img.save(new_path)
+
+    def crop_img(self):
+        """
+        Function that crops the image and pastes it into a blurred background
+        image
+        """
+        img_blurr = Image.open(self.image.path)
+        img_cropped = Image.open(self.image.path)
+        
+        img_blurr = img_blurr.resize((Static.img_size[0], Static.img_size[1]))
+        img_blurr = img_blurr.filter(ImageFilter.BoxBlur(Static.blurr_radius))
+
+        if img_cropped.width > Static.img_size[0] or img_cropped.height > Static.img_size[1]:
+            output_size = (Static.img_size[0], Static.img_size[1])
```

Comment:
Is it not necessary to somehow preserve the aspect ratio? Or does the `thumbnail` method preserve the aspect ration by default?

Reply:
The ratio is preserved. The variable `output_size` maybe rather should be named `max_size`.